include/pref_devices.hpp

```diff
@@ -22,34 +22,32 @@ class QTreeWidget;
 class QBoxLayout;
 class QPxSettings;
 
-class prefDevices : public QWidget
-{
-	Q_OBJECT
+class prefDevices : public QWidget {
+  Q_OBJECT
 public:
-	prefDevices(QPxSettings *iset, QWidget *p=0, Qt::WindowFlags f=0);
-	virtual ~prefDevices();
+  prefDevices(QPxSettings *iset, QWidget *p = 0, Qt::WindowFlags f = 0);
```

Comment:
Codacy found an issue: [Class 'prefDevices' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)